src/SingleBookingPage/bookingItem/AccordionBodyLeg.js

```diff
@@ -95,7 +101,7 @@ const AccordionBodyLeg = (props: LegProps) => {
   );
 };
 
-export default createFragmentContainer(AccordionBodyLeg, {
+export default createFragmentContainer(withRouter(AccordionBodyLeg), {
```

Comment:
why exactly `withRouter`?  Is it not working without it?

Reply:
ouch yes, at the start I used `props.history.push` but I changed to `Link` because was simpler and I forgot to remove `withRouter` 🤦‍♂️